curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/write-higher-order-arrow-functions.english.md

```diff
@@ -71,6 +71,16 @@ console.log(squaredIntegers);
 <section id='solution'>
 
 ```js
-// solution required
+const realNumberArray = [4, 5.6, -9.8, 3.14, 42, 6, 8.34, -2];
+const squareList = (arr) => {
+  "use strict";
+  // change code below this line
+  let squaredIntegers = realNumberArray.filter((num)=>num%2===0&&num>0).map((x)     =>x*x);
```

Comment:
@TheCaffeineDev, sorry, but I don't see any changes to that line. Could you update your solution so that it uses the `arr` parameter passed to `squareList`? It doesn't have to be exactly what I suggested, though I think it should at least use `arr` rather than `realNumberArray`.